src-java/testing/functional-tests/src/main/groovy/org/openkilda/functionaltests/exception/ExpectedHttpClientErrorException.groovy

```diff
@@ -0,0 +1,39 @@
+package org.openkilda.functionaltests.exception
+
+import org.openkilda.messaging.error.MessageError
+import org.springframework.http.HttpStatus
+import org.springframework.web.client.HttpClientErrorException
+
+import java.util.regex.Pattern
+
+class ExpectedHttpClientErrorException {
```

Comment:
For such objects you can use lombok annotations. 

@Value annotation will create constructor, equals and hash code. 

to use lombok here you need to update gradle file `src-java/testing/functional-tests/build.gradle`
```
    implementation 'org.projectlombok:lombok:1.18.22'
```

Reply:
This was from @pkazlenka